### PR TITLE
add torus web provider

### PR DESCRIPTION
**Pull request by dkent600**
New PR based against the dev branch (previous was against competition)

Resolves: https://github.com/daostack/alchemy/issues/1343

* also removes the lint rule for sorting imports which is driving me crazy.  need to fix it in [another issue](https://github.com/daostack/alchemy/issues/1533).

### CODE REVIEW

**Comment by dkent600** [comment]
@orenyodfat FYI Pulling in the latest version of torus does not fix the CSS problem

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "analyze": "NODE_ENV=production NETWORK=main ANALYZE=1 node --max_old_space_size=4096 node_modules/.bin/webpack --config webpack.prod.config.js --progress --colors",
-    "build": "webpack --config webpack.prod.config.js",
+    "build": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --config webpack.prod.config.js",
```
Comment (orenyodfat) [inline package.json]: this does not seems to solve the build issue ?
why this change ?

Comment (dkent600) [reply package.json]: That is an attempt to fix the review app build problem and also the ability to build for prod locally.  It fixes the ability to build for prod locally, but doesn't seem to help with the review app build.

**Comment by orenyodfat** [comment]
Heroku review app should work with this fix 5c93b68
(tested on other test branch) .

**Comment by orenyodfat** [comment]
please use torus.setProvider to support other network than mainnet (default) https://docs.tor.us/api-reference/class#init
https://docs.tor.us/api-reference/ethereum-api#setprovider

File: package.json [hunk 2]
```diff
@@ -84,6 +84,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
+    "@toruslabs/torus-embed": "^1.2.0",
```
Comment (orenyodfat) [inline package.json]: please bump to 1.2.1

File: app.json [hunk 3]
```diff
@@ -3,6 +3,10 @@
   "description": "",
   "scripts": {},
   "env": {
+    "NODE_OPTIONS": {
+      "required": true,
+      "value": "--max_old_space_size=2560"
```
Comment (orenyodfat) [inline app.json]: is this needed ?

**Comment by dkent600** [comment]
@orenyodfat The problem with using rinkeby appears to be a bug in web3modal (formerly web3connect).  I reported it here: https://github.com/Web3Modal/web3modal/issues/130

File: package.json (hunk 2 above)
Comment (dkent600) [reply package.json]: Bumping to 1.2.3, the latest.

Comment (dkent600) [reply package.json]: Also grabbing the latest web3modal, no long in beta.
